groovy/postInit/components/Batteries.groovy

```diff
@@ -223,54 +156,422 @@ mods.gregtech.assembler.recipeBuilder()
         .EUt(16)
         .buildAndRegister()
 
-mods.gregtech.assembler.recipeBuilder()
-        .inputs(metaitem('battery.hull.lv'))
-        .inputs(ore('dustCadmium'))
-        .inputs(ore('dustNickelOxideHydroxide'))
-        .fluidInputs(fluid('potassium_hydroxide_solution') * 100)
-        .outputs(metaitem('battery.re.lv.cadmium'))
+/*
+ * Components
+ */
+
+// Pb anode & PbO2 cathode line
+
+crafting.addShapeless("anode_lead", metaitem('anode.lead'), [
+        metaitem('plateLead'), metaitem('cableGtSingleTin')
+]);
+
+MIXER.recipeBuilder()
+        .inputs(ore('dustLead'))
+        .fluidInputs(fluid('diluted_sulfuric_acid') * 1000)
+        .outputs(metaitem('cathode.lead_paste'))
+        .fluidOutputs(fluid('water') * 1000)
         .duration(100)
         .EUt(16)
         .buildAndRegister()
 
-mods.gregtech.assembler.recipeBuilder()
-        .inputs(metaitem('battery.hull.mv'))
-        .inputs(ore('dustCadmium') * 2)
-        .inputs(ore('dustNickelOxideHydroxide') * 2)
-        .fluidInputs(fluid('potassium_hydroxide_solution') * 200)
-        .outputs(metaitem('battery.re.mv.cadmium'))
+crafting.addShapeless("cathode_lead", metaitem('cathode.lead'), [
+        metaitem('anode.lead'), metaitem('cathode.lead_paste')
+]);
+
+ELECTROLYTIC_CELL.recipeBuilder()
```

Comment:
make this recipe use singleblock electrolyzer

Reply:
```
Error adding GregTech electrolyzer recipe: - Must have at most 1 fluid input, but found 2
```
Guess I'll have to revert it to use the EC

Reply:
I can make electrolyzer to accept 2 fluid inputs.

Reply:
I don't think it matters that much if it's in the EC, given that there exists a crafting recipe which does not require this. However, I'm not a very big fan of needlessly adding fluid inputs to other recipe maps if possible, so I would say to just keep it in the EC for now.